jvm/hermes/src/main/jni/JHermesRuntime.h

```diff
@@ -3,6 +3,7 @@
 #include <hermes/hermes.h>
 #include <fbjni/fbjni.h>
 
+#include "RuntimeScope.h"
```

Comment:
Shouldn't need this include as it'd come from the following `#include "JJSIValue.h"` include